eng/native/build-commons.sh

```diff
@@ -55,7 +55,7 @@ check_prereqs()
         if ! pkg-config openssl ; then
             # We export the proper PKG_CONFIG_PATH where openssl was installed by Homebrew
             # It's important to _export_ it since build-commons.sh is sourced by other scripts such as build-native.sh
-            export PKG_CONFIG_PATH=/usr/local/opt/openssl@1.1/lib/pkgconfig:/usr/local/opt/openssl/lib/pkgconfig
+            export PKG_CONFIG_PATH=`brew --prefix`/opt/openssl@1.1/lib/pkgconfig:`brew --prefix`/opt/openssl/lib/pkgconfig
```

Comment:
shellcheck recommends using `$()` instead of the backtick syntax:

```suggestion
            export PKG_CONFIG_PATH=$(brew --prefix)/opt/openssl@1.1/lib/pkgconfig:$(brew --prefix)/opt/openssl/lib/pkgconfig
```